test/main/perf_compare.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"math"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("should have one argument for a directory with test results")
+		os.Exit(1)
+	}
+	currentFile := os.Args[1] + "/index.html"
+
+	i := 0
+	s := 0
+	global_off := 1.0
+	global_on := 1.0
+	global_cores := 1.0
+	geomean := 0.0
+
+	file, err := os.Open(currentFile)
+	if err != nil {
+		fmt.Println("Can't open", currentFile)
+		os.Exit(1)
+	}
+	defer file.Close()
+
+	var lines []string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+	for i < len(lines) {
+		if strings.Contains(lines[i], "TEST_TYPE_BENCHMARK") && !strings.Contains(lines[i], "log") { // "TestTypeBenchmark"
+			name := strings.Split(lines[i][8:len(lines[i])-20], "_off_") // -21 / -18 -17 ?
+			if len(name[1]) == 2 {
+				name[1] = "  " + name[1]
+			} else if len(name[1]) == 3 {
+				name[1] = " " + name[1]
+			}
+
+			normal_cores := 6
+			if name[0] == "ipsec" {
+				normal_cores = 4
+			} else if name[0] == "nat_fixed_ports" || name[0] == "nat_nonfixed_ports" {
+				// Ask Maria!!!
+			}
+
+			j := i + 1
+			for strings.Contains(lines[j], "Average") == false {
+				j = j + 1
```

Comment:
From here and to the end of function, why not use ++ operator for integer increment?